src/core-react/internal/hooks/useIcon.tsx

```diff
@@ -1,26 +1,38 @@
 import { isValidElement, useMemo } from 'react';
-import { coinbasePaySvg } from '../../../internal/svg/coinbasePaySvg';
+import { applePaySvg } from '../../../internal/svg/applePaySvg';
+import { CoinbasePaySvg } from '../../../internal/svg/coinbasePaySvg';
+import { creditCardSvg } from '../../../internal/svg/creditCardSvg';
 import { fundWalletSvg } from '../../../internal/svg/fundWallet';
 import { swapSettingsSvg } from '../../../internal/svg/swapSettings';
+import { toggleSvg } from '../../../internal/svg/toggleSvg';
 import { walletSvg } from '../../../internal/svg/walletSvg';
 
-export const useIcon = ({ icon }: { icon?: React.ReactNode }) => {
+export const useIcon = ({
+  icon,
+  className,
+}: { icon?: React.ReactNode; className?: string }) => {
   return useMemo(() => {
     if (icon === undefined) {
       return null;
     }
     switch (icon) {
       case 'coinbasePay':
-        return coinbasePaySvg;
+        return <CoinbasePaySvg className={className} />;
```

Comment:
nit: can we return the svg similar to the other cases? 
```suggestion
        return coinbasePaySvg;
```

Reply:
Is it possible to use className on svg? The reason is that I need to pass a color to the svg.

Reply:
Btw you can use `fill="currentColor"` on the SVG and then set `text-blue-400` or whatever you want with Tailwind

Reply:
I would like to have this standard so that when you call `useIcon` it behaves the same regardless of any icon that you pass in. 

I understand you want to change the color of this CoinbasePay icon, but we should have one standard interface for all the SVGs or else there are going to be unintentional cases of icons breaking